providers/common/securitycheck_default.c

```diff
@@ -17,13 +17,13 @@
 #include "internal/nelem.h"
 
 /* Disable the security checks in the default provider */
-int ossl_securitycheck_enabled(void)
+int ossl_securitycheck_enabled(OSSL_LIB_CTX *libctx)
```

Comment:
ossl_unused?

Reply:
Not sure why we need to do that here? We have lots of places where we have unused parameters and don't mark them.

Reply:
> It's good practice to?

Is it? Why?